src/status_im2/contexts/profile/settings/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.contexts.profile.settings.view
+  (:require [oops.core :as oops]
+            [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.reanimated :as reanimated]
+            [react-native.safe-area :as safe-area]
+            [status-im2.common.not-implemented :as not-implemented]
+            [status-im2.contexts.profile.settings.constants :as settings.constants]
+            [status-im2.contexts.profile.settings.header.view :as settings.header]
+            [status-im2.contexts.profile.settings.style :as style]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- f-settings-item-view
+  [data]
+  [quo/category
+   {:list-type :settings
+    :blur?     true
+    :data      data}])
+
+(defn scroll-handler
+  [event scroll-y]
+  (let [current-y (oops/oget event "nativeEvent.contentOffset.y")]
+    (reanimated/set-shared-value scroll-y current-y)))
+
+(def footer
+  [rn/view {:style style/footer-container}
+   [quo/button {:type :danger :icon-left :i/log-out}
+    (i18n/label :t/logout)]])
+
+(defn- settings-view
+  [theme]
+  (let [insets              (safe-area/get-insets)
+        customization-color (rf/sub [:profile/customization-color])
+        scroll-y            (reanimated/use-shared-value 0)]
+    [quo/overlay {:type :shell}
+     [rn/view {:style {:flex 1}}
+      [rn/view
+       {:style (style/navigation-wrapper {:customization-color customization-color
+                                          :inset               (:top insets)
+                                          :theme               theme})}
+       [quo/page-nav
+        {:background :blur
+         :icon-name  :i/close
+         :on-press   #(rf/dispatch [:navigate-back])
+         :right-side [{:icon-name :i/multi-profile :on-press #(rf/dispatch [:open-modal :sign-in])}
+                      {:icon-name :i/qr-code :on-press not-implemented/alert}
+                      {:icon-name :i/share
+                       :on-press  #(debounce/dispatch-and-chill [:open-modal :share-shell] 1000)}]}]]
+      [rn/flat-list
+       {:header                          [settings.header/view {:scroll-y scroll-y}]
+        :data                            settings.constants/settings-items
+        :key-fn                          str
+        :shows-vertical-scroll-indicator false
+        :render-fn                       f-settings-item-view
+        :footer                          footer
+        :scroll-event-throttle           16
+        :on-scroll                       #(scroll-handler % scroll-y)
+        :bounces                         false}]]]))
+
+(defn- internal-view
+  [props]
+  [:f> settings-view props])
+
+(def view (quo.theme/with-theme internal-view))
```

Comment:
Just sharing for awareness, but in my Android emulator the old profile screen loads almost immediately and is capable of showing the opening animation. The new profile screen on the other hand doesn't show the opening animation unless I comment out the flat list rendering the settings items and there's a noticeable delay to open the screen with the flatlist. This shows to me the settings flatlist is too heavy, even though we have just a few instances. We should probably consider making each setting item lighter or if possible not using a flatlist at all (just guesswork).

Something to keep in mind for future PRs for the new profile, not a blocker imo for this PR.

Reply:
It makes sense because `quo/category` is a flat-list as well, and we're nesting multiple flat-lists inside the main one, which is impacting performance.

Reply:
we should never have nested scroll views (flat lists) how do we plan to solve it here?

Reply:
@flexsurfer, I can either bypass the design system and use category item directly within my flat-list, or we can stick to the current approach since this is a small static list and performance should be a big deal IMO

Reply:
Generally, a nested flatlist is not a good idea for large data, but we hardly have 16 items. It wouldn't consume too much mobile resources. One best way to improve the performance is to use `get-item-layout` (probably in the `quo/category` component) as we have fixed item height.